tooling/storybook-addon-apollo-client/src/index.tsx

```diff
@@ -1,7 +1,3 @@
-if (module && module.hot && module.hot.decline) {
-  module.hot.decline();
-}
-
```

Comment:
This was causing the build on PR to fail. Looks like the  `@types/node` is updated from this storybook upgrade, that does not support `hot` on type `NodeModule`.

came across this changeset on storybook's addon kit repo, by removing the code reference seems to have fixed the build.
https://github.com/storybookjs/addon-kit/pull/44